clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1508,6 +1508,27 @@ default DescribeProducersResult describeProducers(Collection<TopicPartition> par
      */
     DescribeProducersResult describeProducers(Collection<TopicPartition> partitions, DescribeProducersOptions options);
 
+    /**
+     * Describe the state of a set of transactionalIds. See
```

Comment:
nit: I would use `transactional ids` rather than `transactionalIds` in the description. There are few other cases in this file.